.nuget/Azure.ApiHub.Sdk.nuproj

```diff
@@ -1,32 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Thanks much for doing this! :-) We still need to deploy as Nuget . so we need corresponding nuspec files, ....

Reply:
We shouldn't need nuspecs, but the proper build step using pack to create the packages.